src/analyzer/protocol/dce-rpc/dce_rpc-auth.pac

```diff
@@ -24,24 +26,43 @@ refine connection DCE_RPC_Conn += {
 			ntlm->Done();
 			delete ntlm;
 			}
+		if ( krb )
+			{
+			krb->Done();
+			delete krb;
+			}
+
 	%}
 
 	function forward_auth(auth: DCE_RPC_Auth, is_orig: bool): bool
 		%{
-		switch ( ${auth.type} )
+		switch ( ${auth.type} )  // https://social.msdn.microsoft.com/Forums/en-US/44212c32-a4f6-4960-8799-0e00821650f4/msrpc-and-dcerpc-security?forum=os_windowsprotocols
 			{
 			case 0x09:
 				if ( ! gssapi )
-					gssapi = zeek::analyzer_mgr->InstantiateAnalyzer("KRB", zeek_analyzer()->Conn());
+					gssapi = zeek::analyzer_mgr->InstantiateAnalyzer("GSSAPI", zeek_analyzer()->Conn());
 				if ( gssapi )
 					gssapi->DeliverStream(${auth.blob}.length(), ${auth.blob}.begin(), is_orig);
 				break;
+			case 0x10:
```

Comment:
You don't happen to have a trace for this that we could include into the test suite, do you?

Reply:
Added tests for both ntlm and netlogon. Did not include kerberos since that doesnt work yet on my pcaps